src/hooks/useApproveCallback.ts

```diff
@@ -91,12 +91,14 @@ export function useApproveCallback(
           approval: { tokenAddress: token.address, spender: spender }
         })
       })
-      .catch((error: Error) => {
-        Sentry.captureException(error, {
-          tags: {
-            action: 'Approve Token'
-          }
-        })
+      .catch((error: any) => {
+        if (error?.code !== 4001) {
```

Comment:
can you add explanations about the error status?

Reply:
Added a comment to the code, usually 4001 code is from an error when the user rejected the transaction